ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: install Rust deps
+  apt:
+    name:
+      - pkg-config
+      - libssl-dev
+    state: present
+
+- name: install rustup in userspace for root
+  shell: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
```

Comment:
Let's avoid curling-to-bash. A simple hash verification would makes sense here:

```
set -eu

RUSTUP_VERSION=1.24.3
RUSTUP_SHA="a3cb081f88a6789d104518b30d4aa410009cd08c3822a1226991d6cf0442a0f8"

curl --proto '=https' --tlsv1.2 -sSf -O \
  https://raw.githubusercontent.com/rust-lang/rustup/${RUSTUP_VERSION}/rustup-init.sh

echo "${OUR_RUSTUP_SHA}  rustup-init.sh" | sha256sum --check --

sh rustup-init.sh \
   --default-toolchain nightly -y \
   --component rust-src
```

Reply:
I'm happy to submit a follow-up PR for this.

Reply:
Makes sense, thanks!